scripts/dependencyCheck.js

```diff
@@ -4,7 +4,7 @@ const depcheck = require('depcheck');
 
 const options = {
   specials: ['bin', 'eslint', 'jest', 'babel', 'webpack'],
-  ignoreDirs: ['build', '.storybook'],
+  ignoreDirs: ['build', '.storybook', '.yarn', 'cypress'],
```

Comment:
I have a quick fix to add here, to display the name of the required dependency: https://github.com/bbc/simorgh/actions/runs/6013335936/job/16310593820

Note that the test output is 'key' which is meaningless. 2 ticks ⏸️

Reply:
✅